src/pykoala/plotting/plot_plot.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import matplotlib.pyplot as plt
+import numpy as np
+# -----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
+def full_path(filename,path, verbose=False):
```

Comment:
This function is not necessary as it can be substituted by `os.path.abspath(path)`, that returns the absolute path of an input filename/relative path.

Reply:
It was in my todo list to do that change it, once I knew about `os.path.join`, but I have not done it yet.

I quickly copy-pasted my version of the file `plot_plot.py` in the code, but as advised arch is not ideal BUT IT WORKS.

I'm deleting this function and using `os.path.join` accordingly.

Reply:
> IT WORKS
In a *nix system, probably yes. Definitely not in Windows.